pkg/ctl/create/utils.go

```diff
@@ -0,0 +1,230 @@
+package create
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/gobwas/glob"
+	"github.com/kris-nova/logger"
+	"github.com/pkg/errors"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"github.com/weaveworks/eksctl/pkg/ami"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha4"
+	"github.com/weaveworks/eksctl/pkg/cfn/manager"
+	"github.com/weaveworks/eksctl/pkg/eks"
+)
+
+// NodeGroupFilter holds filter configuration
+type NodeGroupFilter struct {
+	IgnoreAllExisting bool
+
+	existing sets.String
+	only     []glob.Glob
+	onlySpec string
+}
+
+// NewNodeGroupFilter create new NodeGroupFilter instance
+func NewNodeGroupFilter() *NodeGroupFilter {
+	return &NodeGroupFilter{
+		IgnoreAllExisting: true,
+
+		existing: sets.NewString(),
+	}
+}
+
+// ApplyOnlyFilter parses given globs for exclusive filtering
+func (f *NodeGroupFilter) ApplyOnlyFilter(globs []string, cfg *api.ClusterConfig) error {
+	for _, gs := range globs {
```

Comment:
Can we maybe alias the library so we can actually name one of these `glob` instead of `gs` or `g` to make it less confusing? (Maybe compiledGlob instead of `g`?)
Or name it ngGlob or something like that, perhaps.

Reply:
Indeed, thanks! 👍